tests/Admin/AdminHelperTest.php

```diff
@@ -84,7 +84,11 @@ public function testGetElementAccessPath(): void
         $subObject->expects($this->atLeastOnce())->method('getAnother')->willReturn($sub2Object);
         $sub2Object->expects($this->atLeastOnce())->method('getMoreThings')->willReturn('Value');
 
-        $path = $this->helper->getElementAccessPath('uniquePartOfId_path_to_object_0_another_more_things', $object);
+        $path = $this->getMethodAsPublic('getElementAccessPath')->invoke(
```

Comment:
I would say that since the method is now private, there is no need to test it (instead the test should be done by the method which call this private method, `appendFormFieldElement`).

But since there is currently no `appendFormFieldElement` tests, I think this is ok for now.